### DISCUSSION TITLE
Blood loss from bleeding seemingly cannot be resisted by afflictions

### DISCUSSION THREAD

**Discussion by ghost** [FakeFishGames/Barotrauma#13706] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

Attempted to create an affliction that provided 100% blood loss resistance. Character still received full blood loss from bleeding.

### Reproduction steps

1. Create an affliction that grants any amount of blood loss resistance. I encountered it with the following: 

 ```xml
  <Affliction name="gsrgsrgsgrsg 2 ekectruvrbrbr boogsrgsloo" identifier="humanbloodlossresistance" description="" type="resistance" isbuff="true" causeofdeathdescription="" selfcauseofdeathdescription="" limbspecific="false" indicatorlimb="Torso" maxstrength="1" showiconthreshold="0" showinhealthscannerthreshold="0">
    <icon texture="%ModDir%/Afflictions/new_afflictions.png" sourcerect="0,0,1,1" color="0,0,0,0" origin="0,0" />
    <Effect minstrength="0" maxstrength="1" resistancefor="bloodloss"  minresistance="1.0" maxresistance="1.0"/>
  </Affliction>
```

2. Apply bleeding to a character with aforementioned affliction
3. Observe as said character takes full blood loss:

![image](https://github.com/FakeFishGames/Barotrauma/assets/86113712/18464f9c-84e6-45e0-8062-7b2370b07ec9)

### Bug prevalence

Just once

### Single player or multiplayer?

Other

### -

`quickstart humpback`

### Version

v1.3.0.0/v1.3.0.1

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/pull/5112 (PR in our private repo)
